.github/workflows/build.yml

```diff
@@ -11,6 +11,10 @@ on:
       - main
   pull_request:
 
+env:
+  # Use edge release of buildx (latest RC, fallback to latest stable)
+  SETUP_BUILDX_VERSION: edge
```

Comment:
You also need changes in setup-buildx-action steps: https://github.com/docker/docs/blob/380ca6cc9de0143e462cd36c14f91319de22aece/.github/workflows/build.yml#L24-L26

Like: https://github.com/docker/buildx/pull/2828/files#diff-b6f2546d438fa8cf000fdf5fb9409857d1cd8f24a70fab05725d3501a7a4c753R53-R56

Reply:
ah, sorted